### PR TITLE
BLOCKCHAIN-410 contracts implementation

### PR DESCRIPTION
**Pull request by SzymonZiebaNeti**

### CODE REVIEW

File: src/components/CopyIconWithAddress/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { useRef } from 'react';
+import { useMediaQuery } from 'usehooks-ts';
```
Comment (DorianSternVukotic) [inline src/components/CopyIconWithAddress/index.js:2]: Didnt we already have this ?

Comment (SzymonZiebaNeti) [reply src/components/CopyIconWithAddress/index.js:2]: what exactly you mean

Comment (DorianSternVukotic) [reply src/components/CopyIconWithAddress/index.js:2]: CopyIconWithAddress

Comment (SzymonZiebaNeti) [reply src/components/CopyIconWithAddress/index.js:2]: where?

**Pull request merged**